### PR TITLE
JavaScript Group Capstone

### PR DESCRIPTION
**Pull request by Yulianav09**
### JavaScript Group Capstone

- Fetch Pokemons from API.
- Save likes to the Involvement API.
- Save comments to the Involvement API.
- Counter for Pokemon cards.
- Counter for comments.
- Test for counters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,184 @@
...
+- **Save likes to the Involvement API**
+- **Save comments to the Involvement API**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yulianav09.github.io/Group-capstone/)
```
Comment (Ismailco) [inline README.md:82]: - [x] The live demo link is not working, it redirects to a 404 page kindly make sure to fix this so the web app is deployed properly and the link is working

File: src/modules/displayComment.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
+import { addComment, getCommentsForPokemon } from './API.js';
+import commentsCounter from './commentsCounter.js';
+
+const htmlBody = document.querySelector('body');
+
+// Function to fetch Pokemon data from the PokeAPI
+async function getPokemonDetails(pokemonName) {
```
Comment (Ismailco) [inline src/modules/displayComment.js]: - [x] Kindly make sure to use arrow function instead so your code follows the ES6 best practices

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @Yulianav09, @obibaadoma, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️ Nice colors
✔️ Clean UI
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (Yulianav09) [reply README.md:82]: Hello @Ismailco
Thanks for your comment, for some reason I'm not able to replicate this issue, the page is loading correctly for me.
Could it be a cache Issue?
image

File: src/modules/displayComment.js (hunk 2 above)
Comment (Yulianav09) [reply src/modules/displayComment.js]: Thanks for the suggestion.

Applied to all the conventional functions

**Review by skyv26 — APPROVED** [review]
## Hi @Yulianav09 and @obibaadoma, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 




## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Meaningful **Github** commits ✅ 
- Successfully deployed 🚀 ✅ 
- Used `gh-pages` for deployment ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well-structured project ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**